.github/workflows/full.yml

```diff
@@ -50,10 +45,18 @@ jobs:
         if: ${{ github.event_name == 'pull_request' }}
         working-directory: .github/workflows
         run: sh -x pr_title_validation.sh '${{ github.event.pull_request.title }}'
-
+  yaml_lint:
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/dyrector-io/dyrectorio/alpine-proto:3.17-4
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Run YAML linting
+        run: yamlfmt -lint .
```

Comment:
minor: add line between jobs please

Reply:
The tool, yamlfmt, removes empty lines by default, but we can look into this.

Reply:
Okey, just we are adding a new line between every job.

Reply:
Resolved.